<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DbRunEditorDialog</class>
 <widget class="QDialog" name="DbRunEditorDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>801</width>
    <height>284</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Edit run...</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="configLabel">
       <property name="text">
        <string>Select configuration:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="runLabel">
       <property name="text">
        <string>Select run:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="CCfg::Ui::DatabaseBrowserTree" name="configBrowser"/>
     </item>
     <item row="1" column="2">
      <widget class="CCfg::Ui::DatabaseBrowserTree" name="runBrowser"/>
     </item>
     <item row="1" column="1">
      <widget class="QPushButton" name="addButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>---&gt;</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Close</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>CCfg::Ui::DatabaseBrowserTree</class>
   <extends>QTreeView</extends>
   <header>CCfg/Ui/DatabaseBrowserTree.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DbRunEditorDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>370</x>
     <y>268</y>
    </hint>
    <hint type="destinationlabel">
     <x>386</x>
     <y>280</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DbRunEditorDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>311</x>
     <y>266</y>
    </hint>
    <hint type="destinationlabel">
     <x>312</x>
     <y>285</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
